### ISSUE TITLE
The NumericFilter tooltip relies on jQuery

### ISSUE THREAD

**Issue by tpinne** [adazzle/react-data-grid#477]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/adazzle/react-data-grid/blob/master/CONTRIBUTING.md
```

**Current behavior**
The NumericFilter throws an error because $ is not defined when attachTooltip is invoked.

**Expected/desired behavior**
The tooltip does not depend on jQuery/Bootstrap and uses some react component.

**Reproduction of the problem**
Just use the NumericFilter without jQuery/Bootstrap

**Comment by diogofcunha** [comment]
Hi @tpinne.

Agreed, will accept PR.

**Comment by willgates** [comment]
Fixed in v1.0.51
